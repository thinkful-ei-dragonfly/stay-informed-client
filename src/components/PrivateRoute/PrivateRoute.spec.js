import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import PrivateRoute from './PrivateRoute'

describe(`PrivateRoute component`, () => {
  it('renders a form.PrivateRoute by default', () => {
    const wrapper = shallow(<PrivateRoute />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })

})
