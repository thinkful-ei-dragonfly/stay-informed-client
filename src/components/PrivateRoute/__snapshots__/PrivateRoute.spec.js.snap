// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrivateRoute component renders a form.PrivateRoute by default 1`] = `
<Route
  render={[Function]}
/>
`;
